packages/bpk-component-banner-alert/src/AnimateAndFade.tsx

```diff
@@ -57,7 +57,6 @@ class AnimateAndFade extends Component<Props, State> {
   static defaultProps = {
     animateOnEnter: false,
     animateOnLeave: false,
-    className: null,
```

Comment:
We shouldn't be setting a null value as a className on a react element. It should be defined or undefined.